sale_financial_risk/hooks.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def pre_init_hook(cr):
+    """
+    The objective of this hook is to speed up the installation
+    of the module on an existing Odoo instance.
+    """
+    create_commercial_partner_id_column(cr)
+
+
+def create_commercial_partner_id_column(cr):
+    cr.execute(
+        """
+        ALTER TABLE sale_order_line
+        ADD COLUMN IF NOT EXISTS commercial_partner_id int4;
```

Comment:
Looks like the instances of this repo are running on postgres 9.5, wich doesn't support this syntax